static/app/views/replays/detail/network/index.tsx

```diff
@@ -31,6 +40,15 @@ function NetworkList({replayRecord, networkSpans}: Props) {
     asc: true,
     getValue: row => row[sortConfig.by],
   });
+  const [searchTerm, setSearchTerm] = useState('');
+  const [filters, setFilters] = useState<Filters>({});
+
+  const filteredNetworkSpans = useMemo(
+    () => getFilteredNetworkSpans(networkSpans, searchTerm, filters),
+    [filters, networkSpans, searchTerm]
+  );
+
+  const handleSearch = debounce(query => setSearchTerm(query), 150);
```

Comment:
Debouncing probably isn't _super_ important if the query isn't triggering a network request but still could be useful to save renders. Are you sure this is working though? The function triggers a re-render and recreates the debounced function each time. Might need to memo it